aws-api-appsync/src/main/java/com/amplifyframework/api/aws/SelectionSet.java

```diff
@@ -244,6 +246,12 @@ private SelectionSet wrapPagination(SelectionSet node) {
                 } else {
                     result.add(new SelectionSet(fieldName, null));
                 }
+                for (AuthRule authRule : schema.getAuthRules()) {
+                    if (AuthStrategy.OWNER.equals(authRule.getAuthStrategy())) {
+                        result.add(new SelectionSet(authRule.getOwnerFieldOrDefault(), null));
```

Comment:
```suggestion
                        result.add(new SelectionSet(authRule.getOwnerFieldOrDefault(), Collections.emptySet()));
```

Reply:
I just pushed a change to require `SelectionSet.nodes` to be nonnull, let me know what you think!

Reply:
'Tis a thing of exceptional beauty, @richardmcclellan, thanks!